model-armor/snippets/deleteTemplate.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * Deletes a Model Armor template.
+ *
+ * @param {string} projectId - Google Cloud project ID where the template exists.
+ * @param {string} locationId - Google Cloud location (region) of the template, e.g., 'us-central1'.
+ * @param {string} templateId - Identifier of the template to delete.
+ */
+async function main(projectId, locationId, templateId) {
+  // [START modelarmor_delete_template]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'my-project';
+  // const locationId = 'us-central1';
+  // const templateId = 'my-template';
+
+  const name = `projects/${projectId}/locations/${locationId}/templates/${templateId}`;
+
+  // Imports the Model Armor library
+  const {ModelArmorClient} = require('@google-cloud/modelarmor');
+
+  const options = {};
+  options.apiEndpoint = `modelarmor.${locationId}.rep.googleapis.com`;
+
+  // Instantiates a client
+  const client = new ModelArmorClient(options);
```

Comment:
```suggestion
  const client = new ModelArmorClient({
    apiEndpoint: `modelarmor.${locationId}.rep.googleapis.com`,
  });
```

Client instantiation should be consistent across all snippets.

Reply:
✅